src/app/core/store/shopping/product-reviews/product-reviews.actions.ts

```diff
@@ -0,0 +1,11 @@
+import { createAction } from '@ngrx/store';
+
+import { ProductReviews } from 'ish-core/models/product-reviews/product-reviews.model';
+import { payload } from 'ish-core/utils/ngrx-creators';
+
+export const loadProductReviews = createAction('[Product Review] Load Product Reviews', payload<{ skus: string[] }>());
```

Comment:
we don't need to fetch the reviews of more than one product so we the parameter should be sku of type string